### PR TITLE
fix two tests

### PR DESCRIPTION
**Pull request by moben**
& isn't an operator
2147483648 isn't a valid integer literal, see 1705ae3d8c4d73a65ebe3352b83d3e441f9a5615

### CODE REVIEW

File: tests/semantic/test6.valid.mj [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Test {
 	public static void main(String[] args) { }
 
 	public int flow() {
-		return -2147483648;
+		return -2147483647;
 	}
```
Comment (parttimenerd) [inline tests/semantic/test6.valid.mj]: The number is valid:
Except from the Java 8 Language Specification:
> The decimal literal 2147483648 may appear only as the operand of the unary minus operator - (§15.15.4).

Comment (sgraf812) [inline tests/semantic/test6.valid.mj]: Ouuch... This is getting ugly

Comment (parttimenerd) [inline tests/semantic/test6.valid.mj]: It's probably important to know that `-(2147483648)` isn't valid (Mini)Java.

**Comment by parttimenerd** [comment]
I fixed the only error in one of your test cases in another commit.

File: tests/semantic/test6.valid.mj (hunk 1 above)
Comment (sgraf812) [inline tests/semantic/test6.valid.mj]: 🎉 gotta love MJ grammar

Comment (moben) [inline tests/semantic/test6.valid.mj]: that seems consistent with Java though?

with `-(2147483648)` the operand to the unary minus is `(2147483648)`, not `2147483648`

Comment (parttimenerd) [inline tests/semantic/test6.valid.mj]: Of course. I just pointed it out as it may cause problems for compilers.

**Pull request closed without merging**